the-collab-lab/tcl-55-smart-shopping-list
# Adds about page

## Description
This code adds an about page to give a brief description of the app, link to the developers' websites, and credit the Home page logo. It is accessible by clicking on the List Luxe title.

## Related Issue

Closes #40 

## Acceptance Criteria
- [x] Create the About page following the wireframe in the notes
    - [x] Create a section about the app and how to use it
    - [x] Create a section with a brief bio on each developer that worked on it along with their photos and social links
- [x] Add this page to the home and list pages using a drawer

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
| ✓   | :sparkles: New feature     |

## Updates

### After
![image](https://github.com/the-collab-lab/tcl-55-smart-shopping-list/assets/30507348/9f863e19-e439-4392-9b88-142ef835767a)


## Testing Steps / QA Criteria

In your terminal, use `git checkout tf-jf-about drawer` to create a new branch and switch over.
Use `git pull origin tf-jf-about drawer` to bring the changes from remote to local.
npm start to start a local instance of our app

- Open the drawer from the Home page by clicking on the title and confirm that the Add Item page is displayed within it
- Test the drawer accessibility by opening and closing it using both keyboard and mouse
